search.py

```diff
@@ -299,8 +305,8 @@ def merge_posting_lists(list1, list2, should_perform_merge_positions = False):
     Merges list1 and list2 for the AND boolean operator
     """
     merged_list = PostingList()
-    L1 = list1.size
-    L2 = list2.size
+    L1 = list1.unique_docids
+    L2 = list2.unique_docids
```

Comment:
Should be changed to `len(list<x>.postings)`. We need actual postings length not the unique docids to merge

Reply:
Errrr this one not I make one arh not sure if this will break functionality

Reply:
It'll break. I wrote this function it depends on the acutal posting length not the unique doc id count

Reply:
Ok, so just change 
list1.unique_docids to len(list1.postings)  AND
list2.unique_docids to len(list2.postings) right?

Reply:
If it's the above ^, done!